coral/types/api.d.ts

```diff
@@ -511,6 +553,266 @@ export type components = {
        */
       currentPage?: string;
     };
+    AclRequest: {
+      /**
+       * @description A comment on the request for the approver.
+       * @example Hello, thank you.
+       */
+      remarks?: string;
+      /**
+       * @description This is mandatory if topictype is consumer
+       * @example Group-one
+       */
+      consumergroup?: string;
+      /**
+       * @example [
+       *   "35.239.43.144",
+       *   "35.239.43.145"
+       * ]
+       */
+      acl_ip?: string[];
+      /**
+       * @example [
+       *   "username",
+       *   "username-two"
```

Comment:
```suggestion
       *   "username-two"
       *   "CN=myhost,OU=IS,OU=OU,OU=Services,O=Org"
```

Reply:
These autogenerated types are not part of this PR, I think they are related to the changes @mathieu-anderson  did (`acl_ssl`). My branch may not be up to date here, we should check after merging if we need to update that!